sdk/src/wallet/migration/migrate_3.rs

```diff
@@ -0,0 +1,1227 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::*;
+
+pub struct Migrate;
+
+#[async_trait]
+impl MigrationData for Migrate {
+    const ID: usize = 3;
+    const SDK_VERSION: &'static str = "1.0.0-rc.0";
+    const DATE: time::Date = time::macros::date!(2023 - 07 - DD);
```

Comment:
Sorry this broke it, let's just use today as date
```suggestion
    const DATE: time::Date = time::macros::date!(2023 - 07 - 18);
```